src/components/container/BasicPointsForBadges/BasicPointsForBadges.tsx

```diff
@@ -78,10 +83,10 @@ export default function (props: BasicPointsForBadgesProps) {
     return <div className={"mdhui-basic-points-for-badges"} ref={props.innerRef}>
         {badges !== undefined && points !== undefined &&
             <>
-                <Title order={1} className="mdhui-basic-points-for-badges-points-toward-badge" style={{ color: resolveColor(layoutContext.colorScheme, props.pointsLabelColor) }}>{props.showTotalPoints ? points.toLocaleString() : (props.pointsPerBadge - pointsUntilNextBadge()).toLocaleString()}pts</Title>
+                <Title order={1} className="mdhui-basic-points-for-badges-points-toward-badge" color={props.pointsLabelColor} >{props.showTotalPoints ? points.toLocaleString() : (props.pointsPerBadge - pointsUntilNextBadge()).toLocaleString()}pts</Title>
                 <ProgressBar key={badges.length} // forces re-render to skip "backwards" animation when badges change
                     fillPercent={(props.pointsPerBadge - pointsUntilNextBadge()) / (props.pointsPerBadge * 1.0) * 100}
-                    fillColor={resolveColor(layoutContext.colorScheme, props.progressBarFillColor) || "var(--mdhui-color-primary)"}
+                    fillColor={props.progressBarFillColor}
```

Comment:
I missed the `|| "var(--mdhui-color-primary)"` part when looking at this before.  We may want this to be:

```
fillColor={props.progressBarFillColor || "var(--mdhui-color-primary)"}
```

It looks like there are some default props defined in the ProgressBar with that value, but I'm not sure if that gets overridden with the `undefined` value since we are specifying. 🤔